content/kapp-controller/docs/latest/faq.md

```diff
@@ -0,0 +1,24 @@
+---
+title: FAQ
+---
+
+This documentation covers frequently asked questions for kapp-controller. It is divided into two sections: 
+[Package Author](#package-author) and [Package Consumer](#package-consumer). This is done to organize questions 
+based on how users are working with kapp-controller.
```

Comment:
keep in mind that kapp-controller isnt just for packaging. App CR is big part of it. how do we capture this here?